### PR TITLE
Add duplicator app

### PR DESCRIPTION
**Pull request by raulriera**
Adds the duplicator https://github.com/raulriera/probot-duplicator app to the app gallery.

### CODE REVIEW

**Comment by raulriera** [comment]
Thanks! sure thing

File: _apps/duplicator.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: Duplicator
+description: Create full copies of issues with a single command Edit
```
Comment (bkeepers) [inline _apps/duplicator.md]: Did you mean for the `Edit` to be included here?

File: _apps/duplicator.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+- https://user-images.githubusercontent.com/24159/38773626-522445dc-401f-11e8-9cb4-213bf21e8097.png
+authors:
+- raulriera
+repository: raulriera/probot-duplicator
+host: https://rapid-sousaphone.glitch.me/
+---
+
+# Probot: Duplicator
+
+![Screenshot](https://user-images.githubusercontent.com/24159/38566574-0015d8b6-3cb2-11e8-872f-e9495192581e.png)
```
Comment (bkeepers) [inline _apps/duplicator.md]: This looks like it's from another app.

**Comment by hiimbex** [comment]
The CI issues are due to #167 and #168, not you!

**Pull request merged**